### ISSUE TITLE
Error while updating property blurRadius

### ISSUE THREAD

**Issue by sm2017** [facebook/react-native#16494]
### Is this a bug report?

Yes , I think
The following code is working well on android 6 and 7 but on android 4.2.2 we have a problem

### Have you read the [Contributing Guidelines](https://facebook.github.io/react-native/docs/contributing.html)?
Yes

### Environment

    OS:  Windows 10
    Node: v8.1.3
    Yarn:  1.2.0
    npm:  5.0.3
    Watchman: Not Found
    Xcode:  Not Found
    Android Studio:  2.3.3

    Packages: (wanted => installed)
        react-native: 0.49.3 => 0.49.3
        react: 16.0.0-alpha.12 => 16.0.0-alpha.12

    Target Platform: Android(4.2.2)

### Steps to Reproduce

```js
import React, { Component } from 'react';
import { AppRegistry, Image } from 'react-native';

export default class Bananas extends Component {
  render() {
    let pic = {
      uri: 'https://upload.wikimedia.org/wikipedia/commons/d/de/Bananavarieties.jpg'
    };
    return (
      <Image source={pic} style={{width: 193, height: 110}} blurRadius={0.5}/>
    );
  }
}
```

### Expected Behavior

Blur image

### Actual Behavior

![image](https://user-images.githubusercontent.com/19644997/31875334-9df1ff72-b7da-11e7-9296-27a3fe0ba26d.png)

**Comment by sm2017** [comment]
UP

**Comment by sinewave440hz** [comment]
I am seeing this 7.1.0...

**Comment by sm2017** [comment]
Please fix the issue or at least add a method that let us `blurRadius` is supported or not

**Comment by jamesreggio** [comment]
This may be fixed by #16845.
